_data/events.yml

```diff
@@ -60,6 +65,11 @@
   duration: "4h"
   video: "https://www.youtube.com/embed/Cw7wC8LSZ-Q"
   image: ""
+  gallery:
+  - "assets/images-events/Macropad-3.jpg"
+  - "assets/images-events/Macropad-4.jpg"
+  - "assets/images-events/Macropad-5.jpg"
+  - "assets/images-events/Macropad-6.jpg"
```

Comment:
You are missing pictures 1 & 2 here
Note that the name of the pictures 1 & 2 is DIFFERENT -- COPY PASTE THE FILE NAMES